src/SFA.DAS.AssessorService.ExternalApiDataSync/Startup/Bootstrapper.cs

```diff
@@ -31,7 +43,7 @@ public static void StartUp(ILogger functionLogger, ExecutionContext context)
 
                 configure.For<IAggregateLogger>().Use(_logger).Singleton();
                 configure.For<IWebConfiguration>().Use(configuration).Singleton();
-                configure.For<IDbConnection>().Use(c => new SqlConnection(configuration.SqlConnectionString));
+                configure.For<IDbConnection>().Use(c => ManagedIdentitySqlConnection.GetSqlConnection(configuration.SqlConnectionString, _azureServiceTokenProvider));
```

Comment:
Same issue raised in [this comment](https://github.com/SkillsFundingAgency/das-assessor-service/pull/1057/files#r496557271) and [this comment](https://github.com/SkillsFundingAgency/das-assessor-service/pull/1057/files#r517247524)

Reply:
ExternalApiDataSync has been moved to the Assessor functions repo